### DISCUSSION TITLE
Welcome to Deno Discussions!

### DISCUSSION THREAD

**Discussion by bartlomieju** [denoland/deno#8687] [General]
## 👋 Welcome to Deno Discussions!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about
  * Share ideas for new features

For realtime discussions drop by our Discord: https://discord.gg/deno

**Comment by invisal** [comment]
It is great to be part of this amazing community.

**Comment by hoangtranson** [comment]
I just come to Deno to develop a new project. Normally, I will use expressjs for it. Now I made decision to change to Deno and actually it is great :D Thanks for everyone effort that helped to build Deno :)
